### ISSUE TITLE
Main Datreant Package can't be found on Conda

### ISSUE THREAD

**Issue by jdetle** [datreant/datreant.core#80]
I don't know if datreant has plans for distribution on Conda but it's not currently available for linux 64 channels. 
```
Fetching package metadata .......
Solving package specifications: .
Error: Package missing in current linux-64 channels: 
  - datreant/datreant.core
```

**Comment by kain88-de** [comment]
It isn't on the default channel. Only continuum analytics decides what goes on that channel. We have our own channel though

https://anaconda.org/datreant/packages

**Comment by jdetle** [comment]
Ok thanks!

On Tue, Jul 12, 2016 at 7:29 AM Max Linke <notifications@github.com> wrote:

> Closed #80 <https://github.com/datreant/datreant.core/issues/80>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/datreant/datreant.core/issues/80#event-720218009>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AKcARsyBqKJedN5_S-B55EfCjEy6lzFkks5qU6TVgaJpZM4JKb1A>
> .
>
-- 
Have a wonderful day,
John J. Detlefs
